lib/rubocop/cli.rb

```diff
@@ -159,7 +159,9 @@ def execute_runner(paths)
       display_error_summary(runner.errors)
       maybe_print_corrected_source
 
-      if all_passed && !runner.aborting? && runner.errors.empty?
+      pass_condition = all_passed || @options[:auto_gen_config]
```

Comment:
I would give this a different name, which tells you this is a boolean and is more explicit about what is actually doing, what about `all_pass_or_excluded`?

Reply:
ok `all_pass_or_excluded` seems better